config/stack_noaa.yaml

```diff
@@ -78,10 +78,10 @@ esmf:
   debug: NO
 
 fms:
-  build: NO
-  version: master
+  build: YES
+  version: 2020.04.03
   repo: noaa-gfdl
-  cmake_opts: "-DGFS_PHYS=ON -DLARGEFILE=ON"
+  cmake_opts: "-DGFS_PHYS=ON -D64BIT=ON -DOPENMP=ON"
```

Comment:
I assume this is for 64bit build, we will also have a 32bit real build?

Reply:
`32BIT` is on by default.
We provide both single and double precision libraries in the same installation.
The UFS can link to either based on the choice of the application.
There will be minimal changes in the UFS CMakeLists.txt, namely,  the `if(FMS)` block will look like:
```
if(FMS)
  find_package(FMS REQUIRED COMPONENTS R4 R8)
  if(32BIT)
    add_library(fms ALIAS fms_r4)
  else()
    add_library(fms ALIAS fms_r8)
  endif()
endif(FMS)
```